### DISCUSSION TITLE
Support SMTP

### DISCUSSION THREAD

**Discussion by kentcdodds** [mswjs/msw#751] [Ideas]
**Is your feature request related to a problem? Please describe.**

I'd like to intercept requests made by `nodemailer` on the contact form of my website to test sending emails without actually sending emails 😄 

**Describe the solution you'd like**

I'd love the simplicity of MSW applied to handling sending of emails. `nodemailer` uses SMTP.

**Describe alternatives you've considered**

I could mock `nodemailer` itself, but I can't do that with Cypress very easily and would rather stick with MSW.

**Additional context**

Here's my contact action code: https://github.com/kentcdodds/remix-kentcdodds/blob/dc5b21d34a45ad8fb0da40ca7acc227afea8a3b1/app/utils/send-contact-email.server.ts

Thanks!

**Comment by weyert** [comment]
Sounds like you are asking for TCP/UDP support :)

**Comment by marcosvega91** [comment]
Helloo :),
thanks for opening this issue, we are considering adding this feature in `node-request-interceptor` and the provide a new API here. I'll assign the ticket to myself

**Comment by kentcdodds** [comment]
For what it's worth, I think I'm going to switch to mailgun's REST API as a workaround, so this feature probably won't be necessary for me in the future.

**Comment by kentcdodds** [comment]
I've opened an issue on the interceptors repo: https://github.com/mswjs/interceptors/issues/374

**Reply by kettanaito** [reply]
Let's follow on this in the linked issue.
